.travis.yml

```diff
@@ -1,9 +1,18 @@
 language: generic
+
+git:
+  depth: 5
```

Comment:
Why not simply 1?

Reply:
Just my default setting. If you have parallel jobs and push while not all jobs of previous build are started, it would fail for depth=1.

Reply:
OK